src/llm/__init__.py

```diff
@@ -1,129 +1,27 @@
-import asyncio
-from typing import List
+import logging
+import importlib
 
-from langchain.chains import LLMChain
-from langchain.chat_models import ChatOpenAI
-from langchain.embeddings import OpenAIEmbeddings
-from langchain.memory import ConversationBufferWindowMemory
-from langchain.prompts import (ChatPromptTemplate, HumanMessagePromptTemplate,
-                               MessagesPlaceholder,
-                               SystemMessagePromptTemplate)
-from sqlalchemy import select
+from .default import DialogLLM
+from .abstract_llm import AbstractLLM
+from settings import OPENAI_API_KEY, LLM_CLASS
 
-from learn.idf import categorize_conversation_history
-from llm.memory import generate_memory_instance
-from models import CompanyContent
-from models.db import session
-from settings import (LLM_CONFIG, MODEL_NAME, OPENAI_API_KEY, PROJECT_CONFIG,
-                      PROMPT, VERBOSE_LLM)
 
-CHAT_LLM = ChatOpenAI(
-    openai_api_key=OPENAI_API_KEY,
-    temperature=LLM_CONFIG.get("temperature"),
-    model_name=MODEL_NAME,
-)
-EMBEDDINGS_LLM = OpenAIEmbeddings(openai_api_key=OPENAI_API_KEY)
+def get_llm_class() -> AbstractLLM:
```

Comment:
```suggestion
def get_llm_class() -> Type[AbstractLLM]:
```
https://docs.python.org/3/library/typing.html#the-type-of-class-objects